src/main/resources/config/liquibase/changelog/20220620150000_changelog.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.4.xsd">
+    <!--
+    This changeset adds the display threshold to the exercise_hint table
+    -->
+    <changeSet author="timor-morrien" id="20220620150000">
+        <addColumn tableName="exercise_hint">
+            <column name="display_threshold" type="INTEGER" defaultValueNumeric="3">
```

Comment:
A smaller datatype, e.g. tinyint should be sufficient here. Additionally you can make this column unsigned